cypress/integration/formr-b/formr-b.spec.ts

```diff
@@ -278,6 +278,11 @@ describe("Form R (Part B)", () => {
         cy.checkAndFillSection1(currRevalDate, prevRevalDate);
         cy.get("[data-cy=BtnSaveDraft]").click();
 
+        cy.logout();
+        cy.login();
+        cy.get("[data-cy=BtnMenu]").should("be.visible").click();
+        cy.contains("Form R-b").click();
+
```

Comment:
@babul3012  How are we verifying here that the changes we made before log out still exist after we login again in the application?

Reply:
Added condition to check the gmcNumber. Pushed the changes.